content/docs/components/jupyter.md

```diff
@@ -4,6 +4,18 @@ description = "Using Jupyter notebooks in Kubeflow"
 weight = 10
 +++
 
+## Creating a custom Jupyter image
+You can create your own Jupyter image and use it in your Kubeflow cluster.
+Your custom image needs to meet the requirements created by Kubeflow Notebook Controller. Kubeflow Notebook Controlle  manges the life-cycle of notebooks. 
+Notebook Controller expects the jupyer to be launched upon spawnning a Jupyter notebook. For that you need to set the default command of your image to launch Jupyter. Tha Jupyter launch command needs to be set as follows:
```

Comment:
I think we should make it clear about notebook controller's assumption, and notebook webapp's assumption.

"jupyer to be launched upon spawning" is the webapp's assumption.

Reply:
Done.